### ISSUE TITLE
Wrong code generated for <jpaAnnotations/> Table annotations with KotlinGenerator

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10782]
A formatting error is reported when using `<jpaAnnotations/>` with `KotlinGenerator`

```
11:09:05,377 [INFO] --- jooq-codegen-maven:3.15.0-SNAPSHOT:generate (generate-daos) @ jooq-test-codegen-kotlin ---
java.lang.IllegalStateException: A formatting error has been produced by https://github.com/jOOQ/jOOQ/issues/10196
        at org.jooq.codegen.GeneratorWriter.print(GeneratorWriter.java:162)
        at org.jooq.codegen.GeneratorWriter.print(GeneratorWriter.java:143)
        at org.jooq.codegen.GeneratorWriter.println(GeneratorWriter.java:274)
        at org.jooq.codegen.JavaGenerator.printTableJPAAnnotation(JavaGenerator.java:6734)
        at org.jooq.codegen.JavaGenerator.generatePojo0(JavaGenerator.java:4155)
        at org.jooq.codegen.JavaGenerator.generatePojo(JavaGenerator.java:4124)
        at org.jooq.codegen.JavaGenerator.generatePojo(JavaGenerator.java:4105)
        at org.jooq.codegen.JavaGenerator.generatePojos(JavaGenerator.java:4092)
        at org.jooq.codegen.JavaGenerator.generate(JavaGenerator.java:595)
        at org.jooq.codegen.JavaGenerator.generate(JavaGenerator.java:538)
        at org.jooq.codegen.JavaGenerator.generate(JavaGenerator.java:437)
        at org.jooq.codegen.GenerationTool.run0(GenerationTool.java:879)
        at org.jooq.codegen.GenerationTool.run(GenerationTool.java:233)
        at org.jooq.codegen.GenerationTool.generate(GenerationTool.java:228)
        at org.jooq.codegen.maven.Plugin.execute(Plugin.java:207)
```

The ill formatted Kotlin code can be observed, too (from this [user group discussion](https://groups.google.com/g/jooq-user/c/vwKYEK1CFDI)):

```kotlin
var id: BigInteger?
    set(value) = set(0, value)
@Id
@Column(name = "ID", nullable = false, precision = 19)
    get() = get(0) as BigInteger?
```

----
See also:
https://groups.google.com/g/jooq-user/c/vwKYEK1CFDI

**Comment by lukaseder** [comment]
Actually, there's more than just ill formatted annotations for columns. The POJO code is generated with Java annotation syntax.

**Comment by lukaseder** [comment]
Some more observations:

- Nested annotations must not repeat the `@` symbol, e.g. `@Table(..., uniqueConstraints = [ UniqueConstraint(...) ])`
- Arrays cannot be created using Java's `{ }` literal syntax

**Comment by lukaseder** [comment]
Related: No JPA annotations are being generated on POJOs: https://github.com/jOOQ/jOOQ/issues/10783

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.15.0. Backport scheduled for 3.14.1 (#10785)
